SU-SWS/stanford_basic
# D8 core 1286

# READY FOR REVIEW

# Summary
- Fixing the <h1> spacing on the search page.

# Review By (Date)
- 2/14

# Urgency
- med

# Steps to Test

1. Pull in the branch.
2. Clear caches.
3. Search
4. Verify the <h1> on the search results pages has a margin top that is the same as our other pages.

# Affected Projects or Products
- na

# Associated Issues and/or People
- D8CORE-1286
- @kerri-augenstein 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
